docs/guides/basics/authentication/aws.md

```diff
@@ -9,15 +9,14 @@ APIs that are part of Amazon Web Services (AWS) use a proprietary authentication
 
 The SDK supports two different ways of authenticating with an AWS service: access keys and by assuming a role.
 
-
 ## Access key
 
 The simplest method of authentication requires that the Coda user provides an access key and secret associated with an AWS user. Coda will use these values to sign outgoing requests. Simply set the authentication type to [`AWSAccessKey`][sdk_awsaccesskey] and specify the AWS service you are connecting to.
 
 ```ts
 pack.setUserAuthentication({
   type: coda.AuthenticationType.AWSAccessKey,
-  service: "s3",
+  service: 's3',
```

Comment:
Drive by: The double quotes are correct here, as sample code has a different style guide:

https://coda.io/packs/build/latest/support/contributing/style/#use-double-quotes

I assume that your IDE did this automatically though?